app/src/main/res/values/strings.xml

```diff
@@ -99,21 +99,21 @@
     <string name="cancel">Cancel</string>
     <string name="ok">OK</string>
     <string name="pref_language">Language</string>
-    <string name="pref_language_summary">Change the app language</string>
+    <string name="pref_language_summary">App</string>
     <string name="pref_darkmode">Dark mode</string>
-    <string name="pref_darkmode_summary">Set when dark mode is activated</string>
+    <string name="pref_darkmode_summary">Selects when to use it</string>
     <string name="connection_error">Could not load charging stations</string>
     <string name="retry">Retry</string>
     <string name="filter_open_247">Available 24/7</string>
     <string name="filter_barrierfree">Usable without registration</string>
     <string name="filter_exclude_faults">Exclude chargers with reported faults</string>
     <string name="charge_cards">Payment methods</string>
     <string name="and_n_others">and %d others</string>
-    <string name="pref_map_provider">Map provider</string>
+    <string name="pref_map_provider">Provider</string>
```

Comment:
```suggestion
    <string name="pref_map_provider">Map provider</string>
```
From memory, I think I tried to do away with the other "Provider". What is still "provider" now?